back-end/notifications/notify.py

```diff
@@ -0,0 +1,64 @@
+from django.conf import settings
+from notifications.tasks import (
+    send_single_email_notification,
+    send_mass_email_notification,
+)
+
+
+def send_single_notification(
+    subject: str,
+    recipient: str,
+    content,
+    notification_type: str,
+    description: str,
+    template: str = None,
+    sender: str = None,
```

Comment:
use the default sender email from settings as the default sender

Reply:
resolved